src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -121,7 +121,7 @@ class NUnit extends ConventionTask {
 
     String getNunitName() {
         if (isV35OrAbove) {
-            return "NUnit.${getNunitVersion()}"
+            return "NUnit.Console-${getNunitVersion()}"
```

Comment:
the zip file looks there, am I getting something wrong?
or they just put it back?